src/main/archetype/ui.apps/src/main/content/jcr_root/apps/__appId__/components/adaptiveForm/formcontainer/.content.xml

```diff
@@ -5,4 +5,4 @@
     jcr:title="Adaptive Form Container"
     fd:version="2.1"
     sling:resourceSuperType="core/fd/components/form/container/v2/container"
-    componentGroup=".${appTitle} - Adaptive Form"/>
+    componentGroup="${appTitle} - Adaptive Form"/>
```

Comment:
@barshat7 , is this '.' removed intentionally

Reply:
yes, we will need to unhide to support forms in site